applications/mne_scan/libs/scShared/Management/pluginmanager.h

```diff
@@ -154,6 +155,9 @@ class SCSHAREDSHARED_EXPORT PluginManager : public QPluginLoader
      */
     inline const QVector<IIO*>& getIOPlugins();
 
+Q_SIGNALS:
```

Comment:
we use the `signals` over `Q_SIGNALS` syntax throughout the project

Reply:
Ok. Just read they are just defines. I'll change to enforce a bit of homogeneity.

Reply:
yes, they are just defines. this is only to be in sync with the rest of the project.